scripts/API/java.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+case $1 in
+    setup)
+        mkdir iroha_api_test
+        cd iroha_api_test || exit
+        git clone https://github.com/hyperledger/iroha-java.git
+        cd iroha-java || exit
+        git checkout iroha2-java
```

Comment:
![image](https://user-images.githubusercontent.com/49983831/155068104-ab4da690-0daa-4db0-bb67-feb11309817e.png)

Does the following work?

```suggestion
        git clone https://github.com/hyperledger/iroha-java.git -b iroha2-java
        cd iroha-java || exit
```
The `iroha2-java` branch currently does not exist though